scripts/should_deploy.py

```diff
@@ -18,6 +18,8 @@
 import subprocess
 import sys
 
+from packaging import version
```

Comment:
packaging.version.parse is a third-party utility but is used by [setuptools](https://pypi.python.org/pypi/setuptools) (so you probably already have it installed) and is conformant to the current [PEP 440](https://www.python.org/dev/peps/pep-0440/); it will return a packaging.version.Version if the version is compliant and a packaging.version.LegacyVersion if not. The latter will always sort before valid versions.

Note: packaging has recently been [vendored into setuptools](https://github.com/pypa/setuptools/tree/a94ccbf404a79d56f9b171024dee361de9a948da/pkg_resources/_vendor/packaging).

refer to: https://stackoverflow.com/a/11887885